src/code-formatter.ts

```diff
@@ -64,333 +90,63 @@ export class CodeFormatter {
         return indentedCodeSnippet;
     }
 
-    private adaptCodeSnippet(base64CodeSnippet: string): string {
+    private adaptCodeSnippet(base64CodeSnippet: string): void {
```

Comment:
Ignore the line this finding is attached to. The utility methods above this one like indent, createArray..., createMultiline should be moved to a utils file to have this file as short and concise as possible.